gen-compdocs/generators/doc.go

```diff
@@ -85,10 +97,13 @@ func GenMarkdownCustom(cmd *cobra.Command, w io.Writer, linkHandler func(string)
 	// Note: Files generated for kubeadm tool are snippets of Markdown without a title.
 	// These snippets are included in the corresponding kubeadm pages.
 	if withTitle {
-		if _, err := fmt.Fprintf(w, "---\ntitle: %s\ncontent_type: tool-reference\nweight: 30\n---\n\n", name); err != nil {
+		if _, err := fmt.Fprintf(w, "---\ntitle: %s\ncontent_type: tool-reference\nweight: 30\nauto-generated: true\n---\n\n", name); err != nil {
```

Comment:
This should be okay, though, would check if there is a convention for writing data fields as:
`auto_generated` or `autoGenerated` instead of `auto-generated`

Reply:
Hi @tengqm . I would fix the new field name, otherwise, the changes look good.
/approve
/hold

Reply:
let me put it as auto_generated.

Reply:
:+1: